packages/snaps-cli/src/cmds/build/build.e2e.test.ts

```diff
@@ -18,7 +18,7 @@ describe('mm-snap build', () => {
       command: 'build',
       options: [
         '--src',
-        join(SNAP_DIR, 'src/index.ts'),
+        join(SNAP_DIR, 'src/index.js'),
```

Comment:
This means we aren't E2E testing building TypeScript anymore 🤔 

Not sure that's a good idea.